.gitignore

```diff
@@ -1,5 +1,6 @@
 /build
 /workers/unity/build/
+/test-project/build
```

Comment:
Possibly annoying comment. If this only contains codegen tests can we call it something that reflects that. codegen-test-project or something. test-project is pretty ambiguous.

Reply:
It is an annoying comment 😛